tests/unit_tests/Parser/CMakeLists.txt

```diff
@@ -2,11 +2,12 @@ set (INCLUDE_ROOT ${PROJECT_SOURCE_DIR}/tests/unit_tests/Parser)
 set (SRC_ROOT ${PROJECT_SOURCE_DIR}/tests/unit_tests/Parser)
 
 set (SRC
-    ${SRC_ROOT}/Parser_test.cpp
+    ${SRC_ROOT}/InputParser_test.cpp
 )
 
 create_test(
     TEST_NAME Parser_tests
     TEST_SRC ${SRC}
     TEST_DEPS ${PROJECT_NAME}_parser
+    TEST_DEPS ${PROJECT_NAME}_Core
```

Comment:
In order to add multiple dependencies to a test, you can put them on the same line like so:
```cmake
TEST_DEPS ${PROJECT_NAME}_parser ${PROJECT_NAME}_Core
```